lib/validation.ts

```diff
@@ -22,23 +23,24 @@ import {
   getCwd,
   getExt,
 } from '@hubspot/local-dev-lib/path';
-import { getAccountId, getCmsPublishMode } from './commonOpts';
+import { getCmsPublishMode } from './commonOpts';
 import { logError } from './errorHandlers/index';
 
 export async function validateAccount(
```

Comment:
I noticed that these validation checks weren't working correctly. I'm actually not sure if we should even keep them around. Now that I fixed these I'm starting to see them showing up. For example, I'm getting an error if I have env variables set and I try to run a CLI command without using `--use-env`. But I don't think we need to show an error in that case. The command still works as expected.

I think these checks might be too aggressive, but curious to hear what others think.

Reply:
Hmm yeah that seems unnecessary, you may still want to use your config file after setting env variables. The only thing we really need to check for is using `--account` and `--use-env` at the same time, which we already do